### PR TITLE
fix(windows): disable new hotkey modifier check

### PR DESCRIPTION
**Pull request by mcdurdin**
Disables the new modifier check added in #5189.

### CODE REVIEW

**Comment by darcywong00** [comment]
PR build failed with
>  Error: E2083 Order of fields in record constant differs from declaration
[02:37:29][Step 1/1] UfrmSettingsManager.pas(18) Fatal: F2063 Could not compile used unit 'Keyman.System.Settings.pas

File: windows/src/global/delphi/general/Keyman.System.Settings.pas [hunk 1]
```diff
@@ -401,9 +401,10 @@   TKeymanSettings = class(TObjectList<TKeymanSetting>)
     (
       ID: 'engine.compatibility.old_cached_hotkey_modifier_state';
       Name: SRegValue_Flag_UseCachedHotkeyModifierState;
+      DefaultInt: 1; // 14.0.276: disabling new hotkey modifier check by default
```
Comment (darcywong00) [inline windows/src/global/delphi/general/Keyman.System.Settings.pas:404]: Does this need to go after Description?

Comment (mcdurdin) [reply windows/src/global/delphi/general/Keyman.System.Settings.pas]: Indeed ... my bad 😢

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.276](https://keyman.com/downloads/releases/stable/14.0.276)

**Pull request merged**